src/frontend/screens/AppPasscode/NewPasscode/index.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import { View } from "react-native";
+import { SplashScreen } from "./Splash";
+
+enum ScreenState {
+  splash,
+  setPasscode,
+  reenterPasscode,
+}
+
+export const NewPasscode = () => {
+  const [screenState, setScreenState] = React.useState<ScreenState>(
+    ScreenState.splash
+  );
+
+  function incrementState() {
```

Comment:
nit but maybe rename to `nextStep`? Not sure if addressed in designs but would be good to handle going to previous steps too right?

Reply:
I added the ability to change to a previous step, but I left it as incrementState() because I couldn't think of a word that captures next and back